usaspending_api/download/v2/year_limited_downloads.py

```diff
@@ -1,9 +1,16 @@
-from usaspending_api.awards.v2.lookups.lookups import all_award_types_mappings
+from copy import deepcopy
+from usaspending_api.awards.v2.lookups.lookups import all_award_types_mappings, idv_type_mapping
 from usaspending_api.common.exceptions import InvalidParameterException
 from usaspending_api.download.v2.views import BaseDownloadViewSet
 from usaspending_api.references.models import ToptierAgency
 
 
+# For this endpoint, we need to add IDVs to the all_award_types_mappings.  This
+# will allow us to support IDVs on the backend for bulk downloads as per DEV-1844.
+ALL_AWARD_TYPES_MAPPINGS = deepcopy(all_award_types_mappings)
```

Comment:
Perhaps it would be better to update `all_award_types_mappings` to include `idvs`. It looks like it is used in three places, and another one is for monthly downloads which should probably follow the other downloads.